packages/@react-spectrum/menu/docs/ActionMenu.mdx

```diff
@@ -0,0 +1,261 @@
+
+## Events
+ActionMenu supports the `onAction` prop, which is called when the user presses a menu item. It receives the key of the item as an argument, which can be used to perform the relevant action.
+
+```tsx example
+function Example() {
+  let [action, setAction] = React.useState('');
+
+  return (
+    <>
+      <ActionMenu onAction={setAction}>
+        <Item key="cut">Cut</Item>
+        <Item key="copy">Copy</Item>
+        <Item key="paste">Paste</Item>
+      </ActionMenu>
+      <p>Action: {action}</p>
+    </>
+  );
+}
+```
+
+## Sections
+ActionMenu supports sections in order to group options. Sections can be used by wrapping groups of Items in a `Section` component. Each `Section` takes a `title` and `key` prop.
+
+### Static Items
+
+```tsx example
+<ActionMenu>
+  <Section title="File">
+    <Item key="bold">New</Item>
+    <Item key="underline">Open...</Item>
+  </Section>
+  <Section title="Save">
+    <Item key="left">Save</Item>
+    <Item key="middle">Save As...</Item>
+    <Item key="right">Save All</Item>
+  </Section>
+</ActionMenu>
+```
+
+### Dynamic Items
+
+Sections used with dynamic items are populated from a hierarchical data structure. Similarly to the props on ActionMenu, `<Section>` takes an array of data using the `items` prop.
+
+```tsx example
+function Example() {
+  let openWindows = [
+    {
+      name: 'Reversion',
+      id: 'revert',
+      children: [
+        {id: 11, name: 'Undo'},
+        {id: 12, name: 'Redo'}
+      ]
+    },
+    {
+      name: 'Clipboard',
+      id: 'buffer',
+      children: [
+        {id: 21, name: 'Cut'},
+        {id: 22, name: 'Copy'},
+        {id: 23, name: 'Paste'}
+      ]
+    }
+  ];
+
+  return (
+    <ActionMenu
+      items={openWindows}>
+      {item => (
+        <Section items={item.children} title={item.name}>
+          {item => <Item>{item.name}</Item>}
+        </Section>
+      )}
+    </ActionMenu>
+  );
+}
+```
+
+### Accessibility
+
+Sections without a `title` must provide an `aria-label` for accessibility.
+
+## Complex Menu Items
+
+Items within ActionMenu also allow for additional content used to better communicate options. Icons and descriptions can be added to the `children` of `Item` as shown in the example below. If a description is added, the prop `slot="description"` must be used to distinguish the different `<Text>` elements.
+
+```tsx example
+import {Keyboard, Text} from '@react-spectrum/text';
+<ActionMenu>
+  <Item key="cut" textValue="cut">
+    <Cut size="S"/>
+    <Text>Cut</Text>
+    <Keyboard>⌘X</Keyboard>
+  </Item>
+  <Item key="copy" textValue="copy">
+    <Copy size="S"/>
+    <Text>Copy</Text>
+    <Keyboard>⌘C</Keyboard>
+  </Item>
+  <Item key="paste" textValue="paste">
+    <Paste size="S"/>
+    <Text>Paste</Text>
+    <Keyboard>⌘V</Keyboard>
+  </Item>
+</ActionMenu>
+```
+
+## Props
+
+<PropTable component={docs.exports.ActionMenu} links={docs.links} />
+
+## Visual options
```

Comment:
Mind adding a section for `isQuiet` here?

Reply:
Im also surprised that `isOpen` doesn't show up on the prop list but perhaps out of scope for this PR.